### PR TITLE
fix: Update CI configuration files

### PR DESCRIPTION
**Pull request by ayang64**
This patch updates the Circle CI and Jenkins related configruation files
to allow automated building and testing with Go 1.13.

This also removes any remaning mentions of Go 1.12.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

**Comment by ayang64** [comment]
@gunnaraasen, @jacobmarble -- i tried my best to update the CI related configuration but it seems like it takes a VERY long time for jenkins runs to complete.

any help you can offer would be greatly appreciated.

File: test.sh [hunk 1]
```diff
@@ -123,14 +123,9 @@ case $ENVIRONMENT_INDEX in
...
         ;;
-    4)
-        # go1.12
-        run_test_docker Dockerfile_build_ubuntu64_go1.12 test_64bit --test --junit-report
-        rc=$?
-        ;;
-    "count")
-        echo $ENV_COUNT
-        ;;
+		"count")
```
Comment (jacobmarble) [inline test.sh:126]: Consistent indentation would be nice here. Looks like tabs-vs-spaces.

Comment (ayang64) [reply test.sh:126]: thanks!

Comment (gunnaraasen) [inline test.sh:126]: Looks like this is still set to tabs while the rest of this file uses spaces.

Comment (ayang64) [reply test.sh]: actually fixed this time.

**Pull request closed without merging**